graph/articles/collections.md

```diff
@@ -316,3 +316,148 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
+
+Collections of entity types are generally preferable to collections of structual types because collections of structural types must be updated as a single unit, meaning that they are overwritten entirely by new contents, rather than be updated relative to the existing contents. 
+Sometimes, structural collection properties are added to a type and then scenarios are discovered later that require a collection of entity types.
+Take the following model with an entity type `foo` that has a collection of `bar`s:
+
+```xml
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
+</EntityType>
+
+<ComplexType Name="bar">
+  <Property Name="prop1" Type="Edm.String" />
+  <Property Name="prop2" Type="Edm.String" />
+</ComplexType>
+```
+and a scenario arises that requires, for example, to remove individual `bar`s from the collection. 
+There are two options forward: //// TODO do we want to offer both options, or just one?
+
+### 11.1 Side-by-side collection properties
+
+The model can be updated to have two collections side-by-side:
+```diff
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
++ <NavigationProperty Name="barsAsEntities" Type="Collection(self.barAsEntity)" ContainsTarget="true" />
```

Comment:
We wouldn't really want them to name `barsAsEntities`, would we?  I feel like `Entities` isn't a concept the api consumer necessarily needs to understand.

Reply:
I'm open to other naming conventions, but I do think a naming convention needs to be established for this. If we take your other suggestions and drive workloads to deprecate the old property, then this new property is temporary and can be moved to use the old name if desired. I don't personally have an issue with this name because a client that is wanting to update this collection at least needs to know enough about OData to understand that this is the property that they need to use (instead of the old property). I'm normally opposed to hungarian notation, but in this instance, the difference between the two properties is *literally* the type, and so the naming needs to reflect that.

Reply:
I think the versioning and deprecation doc says, if you can't find a better name, use _v2.  Should we use that for this example?